openapi.yaml

```diff
@@ -12463,6 +12465,45 @@ components:
                 a relative URL, it is relative to the domain you retrieved the
                 Event from.
               example: /v4/support/tickets/11111
+        secondary_entity:
+          type: object
+          readOnly: true
+          description: |
+            Detailed information about the Event's secondary entity, which provides additional information
+            for `linode_boot`, `linode_create`, and `linode_clone` Event actions.
+          x-linode-cli-display: 5
+          properties:
+            id:
+              type: integer
```

Comment:
Not always; images have slugs, for example

Reply:
Will update to be a string.

Reply:
That's probably safer.  This could also come back as an int, but it doesn't look like openapi3 supports this.